Sluggo/Models/Managers/MemberManager.swift

```diff
@@ -40,9 +42,9 @@ class MemberManager {
         
     }
     
-    public func listTeamMembers(completionHandler: @escaping(Result<PaginatedList<MemberRecord>, Error>) -> Void) -> Void{
+    func listFromTeams<T>(page: Int, completionHandler: @escaping (Result<PaginatedList<T>, Error>) -> Void) where T : Decodable, T : Encodable {
```

Comment:
Is there a benefit to making this generic in the implementation here? I'm pretty sure we want to hardcode to MemberRecord, since that's the type we're expecting, but maybe I'm misunderstanding something or this is reducing code elsewhere that I can't see. It makes it sorta hard to see what this is returning, other than intuition that the member call should return a MemberRecord (not sure if something else gets returned from this)

Reply:
This may also be required by the compiler with the interface given, although that would be strange to me if it could not detect generics being specified in an interface. (But I could be misunderstanding how Swift works)

Reply:
I can try to clean this up a little bit, but afaik this is necessary in order to work correctly with the protocol. Protocols aren't 1:1 with other languages so this syntax nastiness might be necessary.